README.md

```diff
@@ -52,6 +52,7 @@ Contributions welcome. Add links through pull requests or create an issue to sta
 - [benxu.dev/blog](https://github.com/AlterionX/benxu-dev) - A relatively simple open source personal blog. Built on [`Seed`](https://seed-rs.org/), [`maud`](https://maud.lambda.xyz), [`Rocket`](https://rocket.rs), and [`Diesel`](https://diesel.rs).
 - [seed-rs.org](https://github.com/seed-rs/seed-rs.org) - Seed's official website.
 - [WeightRS](https://gitlab.com/mkroehnert/weightrs) - Minimalistic and privacy friendly progressive web app for tracking your weight.
+- [planters-rdconf-hackathon-project](https://github.com/ethanboxx/planters-rdconf-hackathon-project) - A basic music composition program. *Note: this is [hackathon](https://rusty-days.org/hackathon/) code so it is very messy.*
```

Comment:
```suggestion
- [Music composer](https://github.com/ethanboxx/planters-rdconf-hackathon-project) - A basic music composition app.
```
Let's make it simple and readable. It should also resolve linter issues.
Notes and links about the hackathon should be in the project README imho.

Reply:
I agree